bluetooth/src/iosMain/kotlin/scanner/Scanner.kt

```diff
@@ -193,4 +200,34 @@ actual class Scanner internal constructor(
             )
         }
     }
+
+    class ScanSettings private constructor(
+        private val allowDuplicateKeys: Boolean,
+        private val solicitedServiceUUIDsKey: List<UUID>?
+    ) {
```

Comment:
Option to pass scan settings, default value for iOS allowDuplicateKeys is false. In the builder we set it to true to match Android default behavior which right now can not be changed to singe discovery (needs a separate feature for a common scan options API)

Reply:
for future reference: https://github.com/splendo/kaluga/issues/506